packages/engine/stdlib/src/py/hstd/init.py

```diff
@@ -0,0 +1,105 @@
+"""
+Initialization utility functions.
+"""
+import math
+import random
+from copy import deepcopy
+from dataclasses import dataclass
+from typing import Optional, List, Union, Callable, Mapping
+
+from .agent import AgentState
+
+AgentFunction = Callable[[], AgentState]
```

Comment:
I could be missing something -- can you explain why `AgentFunction` is required?

Reply:
Well I use it in the type definition of the AgentTemplate. Are you saying I should just do 
`AgentTemplate = Union[AgentState, Callable[[], AgentState]]`?

Reply:
I think it's okay as it is. It looks like the init functions can take an agent object, or a function which returns an agent object, which makes sense. Maybe writing a comment explaining the type's purpose would be useful for future reference.